src/concept/mod.rs

```diff
@@ -44,6 +44,21 @@ pub enum Concept {
     Attribute(Attribute),
 }
 
+impl Concept {
+    pub fn label(&self) -> String {
```

Comment:
```suggestion
    pub fn type_label(&self) -> String {
```

Reply:
Why does this clone? Why not return `&str`?

Reply:
Right, I suppose we can't. In this case this should be renamed to smth like `type_label_cloned()`.